pkg/builder/util.go

```diff
@@ -63,7 +63,7 @@ func ImageConfigStr(labels map[string]string,
 // DefaultPathEnvUnix is unix style list of directories to search for
 // executables. Each directory is separated from the next by a colon
 // ':' character .
-const DefaultPathEnvUnix = "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/opt/conda/bin:/usr/local/julia/bin"
+const DefaultPathEnvUnix = "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/opt/conda/bin:/opt/conda/envs/envd/bin/:/usr/local/julia/bin"
```

Comment:
`bash -c` doesn't work. I guess it's because the conda env is added by `source /opt/conda/bin/activate envd`

Reply:
Any suggestions? @gaocegege

Reply:
LGTM